engine/src/main/java/org/hibernate/validator/resourceloading/AggregateResourceBundle.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.resourceloading;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.Enumeration;
+import java.util.Locale;
+import java.util.Properties;
+import java.util.ResourceBundle;
+
+/**
+ * Inspired by <a href="http://stackoverflow.com/questions/4614465/is-it-possible-to-include-resource-bundle-files-within-a-resource-bundle">this</a>
+ * Stack Overflow question.
+ *
+ * @author Hardy Ferentschik
+ */
+public class AggregateResourceBundle extends ResourceBundle {
```

Comment:
Is there any difference to the inner class of the same name in `PlatformResourceBundleLocator`? Can this be merged to one class? If not, different names would be nice to make the different semantics apparent.